uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/absence-tab/components/annual-entitlement-change-log.component.spec.js

```diff
@@ -165,7 +165,7 @@ define([
 
             beforeEach(function () {
               absenceTypeCalculationUnits = _.pluck(ctrl.absenceTypes,
-                'calculation_unit.name');
+                'calculation_unit_name');
```

Comment:
why is this kept in separate line?
Can't we write `absenceTypeCalculationUnits = _.pluck(ctrl.absenceTypes,'calculation_unit_name');` ?

Reply:
I'm trying to avoid going over the 80 characters limit per line, which is not part of the styleguide, but it's a good rule to follow.

But yes, in this case I think it would make more to have these in a single line.